GVFS/GVFS/Program.cs

```diff
@@ -94,14 +98,24 @@ public static void Main(string[] args)
                             }
 
                             verb.Execute();
-                            Environment.Exit((int)ReturnCode.Success);
+                           Exit((int)ReturnCode.Success);

```

Comment:
Nit: `verb` and `Exit` are not lining up vertically, could you fix the spacing of the code in this block?